contrib/charts/navigator/templates/rbac.yaml

```diff
@@ -95,6 +95,10 @@ items:
   - apiGroups: ["apps"]
     resources: ["statefulsets"]
     verbs:     ["get", "list", "watch", "update", "create", "delete"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs:     ["*"]
+    resourceNames: ["navigator-controller"]
```

Comment:
❓ What does this do?

Reply:
This allows navigator-controller to modify Endpoints resources with the name `navigator-controller`. This is the resource used for leader election 😄